src/ClientAwareInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Phergie (http://phergie.org)
+ *
+ * @link http://github.com/phergie/phergie-irc-bot-react for the canonical source repository
+ * @copyright Copyright (c) 2008-2015 Phergie Development Team (http://phergie.org)
+ * @license http://phergie.org/license New BSD License
+ * @package Phergie\Irc\Bot\React
+ */
+
+namespace Phergie\Irc\Bot\React;
+
+use Phergie\Irc\Client\React\Client;
```

Comment:
This should use `ClientInterface` rather than `Client` so that other client implementations can be used.